### ISSUE TITLE
RSS mauvais sujet

### ISSUE THREAD

**Issue by naldir** [swergas/GentilsVirusSite#3]
Le flux RSS devrait être un flux équivalent aux newsletters, pas un flux des commits github auquels personne n'est intéressé

**Comment by gentilvirus** [comment]
Bonjour @naldir 
Je te propose de venir discuter de tes propositions sur notre [groupe Loomio](https://www.loomio.org/d/MmsLF1a0/) car c'est là que les décisions sur les modifications à apporter au site se prennent.

**Comment by naldir** [comment]
yep, j'e sais bien mais personne ne m''accepte sur loomio donc je ne peux pas participer

**Comment by gentilvirus** [comment]
Désolé pour la réponse tardive. Ce n'est pas normal que personne ne t'accepte sur le groupe!
Je ne suis pas coordinateur du sous-groupe gentilvirus.org sur Loomio, mais je peux t'accepter sur le [groupe parent](https://www.loomio.org/g/4GvkSN9U/gentils-virus-officiel-graine-de-democratie) si tu introduis une demande ( ton pseudo c'est/ce sera aussi @naldir ) ?
